src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -448,59 +448,46 @@ void OTARequestor::NotifyUpdateApplied(uint32_t version)
     ConnectToProvider(kNotifyUpdateApplied);
 }
 
-CHIP_ERROR OTARequestor::GetDefaultOtaProviderList(AttributeValueEncoder & encoder)
```

Comment:
Now that we can get an iterator to the list, there is no need to have OTARequestor class encode. The server can do this directly.